$NetBSD: patch-apps_Makefile.am,v 1.2 2016/01/05 12:51:20 sevan Exp $

Remove install-exec-hook as we handle things with CONF_FILES during post-install
phase

--- apps/Makefile.am.orig	2016-01-04 01:06:40.000000000 +0000
+++ apps/Makefile.am
@@ -90,21 +90,6 @@ EXTRA_DIST += openssl.cnf
 EXTRA_DIST += x509v3.cnf
 EXTRA_DIST += CMakeLists.txt
 
-install-exec-hook:
-	@if [ "@OPENSSLDIR@x" != "x" ]; then \
-		OPENSSLDIR="$(DESTDIR)/@OPENSSLDIR@"; \
-	else \
-		OPENSSLDIR="$(DESTDIR)/$(sysconfdir)/ssl"; \
-	fi; \
-	mkdir -p "$$OPENSSLDIR/certs"; \
-	for i in cert.pem openssl.cnf x509v3.cnf; do \
-		if [ ! -f "$$OPENSSLDIR/$i" ]; then \
-			$(INSTALL) -m 644 "$(srcdir)/$$i" "$$OPENSSLDIR/$$i"; \
-		else \
-			echo " $$OPENSSLDIR/$$i already exists, install will not overwrite"; \
-		fi \
-	done
-
 uninstall-local:
 	@if [ "@OPENSSLDIR@x" != "x" ]; then \
 		OPENSSLDIR="$(DESTDIR)/@OPENSSLDIR@"; \
